docs/예시/봇 서버 수 업데이트하기.md

```diff
@@ -2,4 +2,12 @@
 
 해당 페이지에서는 봇의 서버 수를 확인하는 코드를 포함하고 있습니다.
 
-## SDK를 사용하여 재작성 예정
+## Python
+
+```py
+```
```

Comment:
```suggestion
```py
import koreanbots
import asyncio
kb = koreanbots.Koreanbots()
data = asyncio.run(kb.botinfo(653534001742741552))
print(data.servers)

```
```

Reply:
The description was wrong. Could you check it again?